src/resources/platform_handlers/tensorflow_savedmodel/model.py

```diff
@@ -1,536 +0,0 @@
-# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Are these models now moved to PyTorch/TF backends?

Reply:
Discussed offline. These platform handlers will become Python-based backends stored in other repositories.